package.json

```diff
@@ -98,7 +98,7 @@
     "docs": "yarn build:docs && jsdoc -c ./config/.jsdoc.json",
     "test": "yarn build:test && yarn test:node && yarn test:integration && yarn test:browser",
     "test:e2e": "./test/e2e/initialize.sh && yarn _nyc mocha --recursive 'test/e2e/src/test-*.js'",
-    "test:node": "yarn _nyc mocha --recursive 'test/unit/**/*.js'",
+    "test:node": "NODE_OPTIONS=$(test ! $(node -v | awk -F'.' '{printf substr($1,2)}') = '18' && echo '--no-experimental-detect-module') yarn _nyc mocha --recursive 'test/unit/**/*.js'",
```

Comment:
would it be worth stating the node engine version requirement explicitly here in package? 
```
{
  "engines": {
    "node": ">=18.0.0"
  }
}

```

This may provide some context/bounds to the version 18 check here, it it ok if version <= 17?
 
Maybe there are reasons not having node runtime version, and am just not aware.

Reply:
Nice call! I didn't know you could enforce it this way. Done in 0d85200e :call_me_hand: